nephilimV5-vf/nephilim_V5.json

```diff
@@ -0,0 +1,8 @@
+{
+    "html": "nephilim-legende-vf.html",
+    "css": "nephilim-legende-vf.css",
+    "authors": "Rwan",
+    "roll20userid": "897499",
+    "preview": "previewneph.png",

```

Comment:
"previewneph.png" will fail because your image is "previewneph.PNG". The names need to match exactly including caps.

Reply:
done ; i updated the json file